src/test/java/org/jabref/logic/bibtex/comparator/EntryComparatorTest.java

```diff
@@ -4,131 +4,128 @@
 import org.jabref.model.entry.field.InternalField;
 import org.jabref.model.entry.field.StandardField;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class EntryComparatorTest {
+    private BibEntry e1;
+    private BibEntry e2;
+    @BeforeEach
+    void setUp() {
+            e1 = new BibEntry();
+            e2 = new BibEntry();
+    }
```

Comment:
I know, this was lots of effort. Please revert this change. The tests were more readable before the update.